homeassistant/components/tractive/const.py

```diff
@@ -26,9 +26,16 @@
 CLIENT = "client"
 TRACKABLES = "trackables"
 
+TRACKER_ACTIVITY_STATUS_UPDATED = f"{DOMAIN}_tracker_activity_updated"
 TRACKER_HARDWARE_STATUS_UPDATED = f"{DOMAIN}_tracker_hardware_status_updated"
 TRACKER_POSITION_UPDATED = f"{DOMAIN}_tracker_position_updated"
-TRACKER_ACTIVITY_STATUS_UPDATED = f"{DOMAIN}_tracker_activity_updated"
+TRACKER_SWITCH_STATUS_UPDATED = f"{DOMAIN}_tracker_wellness_updated"
```

Comment:
```suggestion
TRACKER_SWITCH_STATUS_UPDATED = f"{DOMAIN}_tracker_switch_updated"
```

Probably a copy/paste error

Reply:
I just sorted these constants, TRACKER_ACTIVITY_STATUS_UPDATED is on line 29.

Reply:
Sorry, I was blind yesterday 😂